extensions/gitpod-remote/src/extension.ts

```diff
@@ -18,6 +20,9 @@ export async function activate(context: vscode.ExtensionContext) {
 
 	registerTasks(gitpodContext);
 	installInitialExtensions(gitpodContext);
+	getSupervisorPorts(gitpodContext).then(portList => configureMachineSettings(gitpodContext!, portList).then(() => tunnelPorts(portList)));
```

Comment:
if i change .gitpod.yml i need to restart a workspace, can we align with web to have consistent experience between both?